### DISCUSSION TITLE
Exit code: 1

### DISCUSSION THREAD

**Discussion by Drummer19821982** [aclap-dev/video-downloadhelper#247] [Help and Q&A]
### Your question

Por favor, depois d ealguns dia não consigo mais baixar videos. Sempre apresenta o mesmo erro "Error: Exit code: 1

    at rs.receive (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:13157)
    at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42624"

### Addon & CoApp version and details

Video DownloadHelper
Versão 8.2.1.1 (stable)
Localização do Navegador: pt-BR
Construção de produção
Construído em 2024-02-19
Opções de Construção: browser=chrome, linuxlic=false, noyt=true, target=google

Plataforma Win x86-64
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36

**Comment by Mattmjs** [comment]
Companion app version is missing and share a link to where the problem is occurring.

**Reply by Drummer19821982** [reply]
Auxiliar do Aplicativo encontrado VdhCoApp 2.0.19

https://cursosconstruir.com.br/lesson/detail/92/5049

Já baxei diversos videos desse site. Possuo acesso, pois é um curso pago. Depois de um tempo não consegui baixar mais.

**Reply by Mattmjs** [reply]
Ok so it requires a login, could you try in Firefox to see if it is the same or limited to chrome.
If you get the same result, share the master manifest and media manifest links found in the details page in VDH.

**Reply by Drummer19821982** [reply]
Mesmo erro no Firefox

Exit code: 1

moz-extension://ca1e9574-0a6e-4a2c-afc5-97b92d7bd1da/background/main.js:1
receive@moz-extension://ca1e9574-0a6e-4a2c-afc5-97b92d7bd1da/background/main.js:1:13157
Zp</callCatchAppNotFound/</<@moz-extension://ca1e9574-0a6e-4a2c-afc5-97b92d7bd1da/background/main.js:1:42626

**Reply by Drummer19821982** [reply]
chunked | hls
-- | --
core_media | {"builder":"NakedHls""protocol":"hls""duration":1678.8"container":{"name":"Mp4""extension":"mp4""audio_only_extension":"mp3""defacto_codecs":{"audio":{}"video":{}}"supported_video_codecs":[0:"H264"1:"H265"2:"Av1"3:"MP4V"4:"MPEG2"5:"unknown"]"supported_audio_codecs":[0:"Opus"1:"MP3"2:"FLAC"3:"AAC"4:"unknown"]"mimetype":{}}"av":{"audio":false"video":{"codec":{"name":"H264""type":"video""mimetype":{}"defacto_container":"Mp4"}"fps":{}"dimensions":{}"quality":{}"bitrate":{}}}}
created | 1709640271651
descrPrefix | Transmissão HLS
duration | 1678.8
extension | mp4
group | grp-19844420
headers | [0:{"name":"sec-ch-ua""value":""Chromium";v="122", "Not(A:Brand";v="24", "Google Chrome";v="122..."}1:{"name":"sec-ch-ua-mobile""value":"?0"}2:{"name":"User-Agent""value":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KH..."}3:{"name":"sec-ch-ua-platform""value":""Windows""}4:{"name":"Accept""value":"*/*"}5:{"name":"Sec-Fetch-Site""value":"same-origin"}6:{"name":"Sec-Fetch-Mode""value":"cors"}7:{"name":"Sec-Fetch-Dest""value":"empty"}8:{"name":"Referer""value":"https://iframe.mediadelivery.net/embed/61261/4625e1ea-d8c9-449d-..."}9:{"name":"Accept-Encoding""value":"gzip, deflate, zstd"}10:{"name":"Accept-Language""value":"pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7"}]
id | nakedhls:19844420
isPrivate | false
mediaManifest | https://iframe.mediadelivery.net/4625e1ea-d8c9-449d-a0bc-8ed98eeb6990/1280x720/video.drm?contextId=9eee619f-30cb-444c-affa-3929c1d083ee
opStartDate | 1709640314946
operation | null
originalId | network-probe:19844420
pageUrl | https://cursosconstruir.com.br/lesson/detail/92/5066
referrer | https://cursosconstruir.com.br/lesson/detail/92/5066
running | 0
status | inactive
tabId | 2029400600
title | Cursos Construir - Treinamentos em Softwares para Engenharia, Arquitetura e Design de Interiores.
topUrl | https://cursosconstruir.com.br/lesson/detail/92/5066

**Comment by nzy** [comment]
Same here.

Exit code: 1

moz-extension://b9ce8235-0f98-406a-8b84-374ecc8c978a/background/main.js:1
receive@moz-extension://b9ce8235-0f98-406a-8b84-374ecc8c978a/background/main.js:1:13157
Zp</callCatchAppNotFound/</<@moz-extension://b9ce8235-0f98-406a-8b84-374ecc8c978a/background/main.js:1:42626

https://www.youtube.com/watch?v=FU_04xDi8fY&ab_channel=TimL.T.I

**Reply by GrampaWildWilly** [reply]
I am running Windows 11 Pro 64-bit version 23H2 build 22631.3235, Firefox 123.0 64-bit, licensed VDH 8.2.2.4a1 beta, CoApp 2.0.19.  I do not work for VDH.  I am just another user.

I went to your link & this is what I got, without clicking the play button in the player.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/bf70d8ca-c397-4a3b-a934-efd725c0d6cf)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/8c10e08f-3193-416a-9403-0f5fa4df1bca)

It takes 2 images because the VDH menu was long.  I don't know why there were 2 complete sets of variants displayed.  They appear to be duplicates.  I think I may have some settings configured in such a way that VDH recognizes more things than are really necessary.  I didn't use the second set of variants anyway.

Here are the results I got:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/c9058370-7306-422e-b7a7-711096c8067a)
![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/c25a1a81-eb72-4c91-8849-03c5464df752)

Here is one of them playing in VLC:

![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/df347cf3-664f-4d93-a253-bed0393775ff)

The two videos were identical.

So my usual advice applies. Get on the latest releases of everything. That means the VDH beta. You can find that -- and many other useful things -- by clicking the Wiki link at the top of this page. The Wiki page shows a Table of Contents down the right side. You would do well to look at all of the chapters, but for the moment, just go to the chapter on Beta versions. You should be able to take it from there.

**Comment by Mattmjs** [comment]
The media manifest has in there video.drm , was there another case of this
somewhere ?

It looks like some protection to prevent downloading.

**Reply by Drummer19821982** [reply]
It's only happening on this site. I've already tested it on others and VDH works perfectly.

I ended up testing VDH on another computer, without registering the license. Downloaded the video. Then I inserted the license I purchased and it continued with the same error reported previously.
Please note that you had already downloaded approximately 80 videos from this site.

**Reply by GrampaWildWilly** [reply]
That does seem rather odd.  You could try to reload the extension after you put in your license.  That's in the VDH settings under the More tab.  But you've probably closed all your browser windows since then so this advice is probably moot.

To test out Matt's theory, go into the browser settings & disable DRM support.  Then reload the video page.  Then try to play the video.  If the browser pops up an error about requiring DRM to play some content on the page, then you're out of luck.  VDH does not download content protected by DRM.  Mind you, when I've hit DRM-protected content, VDH in fact does download it.  But the video image is scrambled so it's useless.

**Reply by Mattmjs** [reply]
There was another case matching this one and the solution might work https://github.com/aclap-dev/video-downloadhelper/discussions/29#discussioncomment-8359192
